setup.cfg

```diff
@@ -57,7 +57,8 @@ install_requires =
     tabulate
     rich~=13.5
     # For automatic login
-    aiohttp~=3.8
+    # 3.9 aiohttp introduced warnings in Python 3.11 and 3.12
+    aiohttp==3.8.*
```

Comment:
**thought**: I've noticed this too. I started fixing it but I'll need another pass: https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/343